src/list-data/request-action-mapping.ts

```diff
@@ -103,6 +103,7 @@ export const BusinessLookupEntityTypes = [
   EntityTypes.CR,
   EntityTypes.UL,
   EntityTypes.XCR,
+  EntityTypes.CP,
```

Comment:
Sure, but please add in alpha order :)

Reply:
What about XCP?

Reply:
What about SP and GP?

Reply:
What about the other entity types listed in the flow diagram, that might be returned by the business lookup/search?

Reply:
I am not sure about what the codes are for below entity types but these are some other entity types listed in the flow diagram for Name Change. I am going through other request actions as well to see if any missing
- DBA
- Private Act
- Credit Union
- Parish

Reply:
LGTM. Ketaki, OK to resolve this conversation?